pkg/interfaces/contracts/standalone-utils/IWeightedLPOracleFactory.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.24;
+
+import { AggregatorV3Interface } from "@chainlink/contracts/src/v0.8/shared/interfaces/AggregatorV3Interface.sol";
+
+import { IWeightedPool } from "../pool-weighted/IWeightedPool.sol";
+import { IWeightedLPOracle } from "./IWeightedLPOracle.sol";
+
+/**
+ * @title Weighted LP Oracle Factory
+ * @notice Factory contract for deploying and managing lp weighted pool oracles.
+ */
+interface IWeightedLPOracleFactory {
```

Comment:
Let's also add a getter that returns true given an oracle address if it comes from this factory, and false otherwise (similar to `isPoolFromFactory`).

Unlike the factories, I don't think we need to store the list of deployed oracles in this case.